packages/shex-webapp/doc/shex-simple.html

```diff
@@ -317,7 +317,7 @@ <h2 class="status"> </h2>
                 <li><a href="#fixedMap-tab">Fixed Map</a></li>
               </ul>
               <!-- <div id="textMap-tab"> -->
-                <textarea id="textMap" style="width:100%; height:100%;"></textarea>
+                <textarea id="textMap" style="width:100%; height:100%;" placeholder="&lt;SomeFocusNode&gt;@START"></textarea>
```

Comment:
What is this for? Why not set it either directly or just in the JS?

Reply:
I don’t understand what you mean. What should be set directly?

Reply:
It seems to me that you set the placeholder attribute here and also via JS. I wondered why you did that?

Reply:
The JS override is only when a certain option is set, in which case the default placeholder (set here) isn’t appropriate.